src/store/personalia/reducer.ts

```diff
@@ -61,15 +101,25 @@ const personaliaReducer: Reducer<PersonaliaState, PersonaliaActions> = (state =
 
 function* hentPersonalia(action: HentPersonaliaAction) {
     try {
-        const response = yield call( () => fetchPersonaliaData(action.fnr));
+        const response = yield call(() => fetchPersonaliaData(action.fnr));
         yield put(hentPersonaliaSuccess(response));
     } catch (e) {
         yield put(hentPersonaliaError(e));
     }
 }
 
+function* hentInnloggetNivaa4(action: HentHarbruktNivaa4Action) {
```

Comment:
Lurer på om nivå4 kanskje skal få en egen reducer...

Reply:
Tror ikke det hadde vært for dumt. Men syns heller ikke at dette er for ille.

Reply:
Ente med å la det være sånn iom samme backend konsekvent bruker samme reducer i repoet